web/src/app/schedules/ScheduleOverrideForm.js

```diff
@@ -1,24 +1,25 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import p from 'prop-types'
 import { FormContainer, FormField } from '../forms'
 import {
   Grid,
   InputAdornment,
   IconButton,
   Typography,
-  withStyles,
+  makeStyles,
 } from '@material-ui/core'
+import { useQuery } from 'react-apollo'
 import { ScheduleTZFilter } from './ScheduleTZFilter'
-import { connect } from 'react-redux'
+import { useSelector } from 'react-redux'
 import { urlParamSelector } from '../selectors'
-import { DateRange, ChevronRight, ChevronLeft } from '@material-ui/icons'
+import { DateRange } from '@material-ui/icons'
 import { DateTimePicker } from '@material-ui/pickers'
 import { DateTime } from 'luxon'
 import { UserSelect } from '../selection'
-import Query from '../util/Query'
 import gql from 'graphql-tag'
 import { mapOverrideUserError } from './util'
 import DialogContentError from '../dialogs/components/DialogContentError'
+import _ from 'lodash-es/lodash.default'
```

Comment:
We should fix it so they are all the same -- it will make searching and the transition from `lodash-es` to `lodash` easier too.

Reply:
Yep, IDE auto-imports strike again, thanks for catching